tools/rna_tools/cmsearch_deoverlap/cmsearch-deoverlap.xml

```diff
@@ -0,0 +1,92 @@
+        <data name="matches_sort" format="tabular" label="Sorted matches">
+            <filter>dirty</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="2">
+            <param name="clan_information" value="ribo.claninfo" />
+            <param name="cmsearch_matches" value="1.cmsearch.tblout" />
+            <param name="dirty" value="--dirty"/>
+            <output name="deoverlapped_matches" file="1.cmsearch.tblout.deoverlapped" />
+            <output name="matches_sort" file="1.cmsearch.tblout.sort" />
+        </test>
+        <test expect_num_outputs="1">
+            <param name="clan_information" value="ribo.claninfo" />
+            <param name="cmsearch_matches" value="1.cmsearch.tblout" />
+            <param name="dirty" value=""/>
+            <output name="deoverlapped_matches" file="1.cmsearch.tblout.deoverlapped" />
+        </test>
+    </tests>
+    <help><![CDATA[
+
+.. class:: infomark
+
+**What this tool does**
+
+This tool removes lower-scoring overlaps from cmsearch results. Given information about the clans and the cmsearch tbl output this tool returns a filtered cmsearch tbl output.
+
+**clan.info:**
+
+CL00111	SSU_rRNA_bacteria SSU_rRNA_archaea SSU_rRNA_eukarya SSU_rRNA_microsporidia SSU_trypano_mito
+
+CL00112	LSU_rRNA_archaea LSU_rRNA_bacteria LSU_rRNA_eukarya LSU_trypano_mito 5_8S_rRNA
+
+CL99999 5S_rRNA mtPerm-5S
+
+**cmsearch.tblout:**
+
+contig--2249925      -         SSU_rRNA_bacteria    RF00177   hmm       19      914        2      898      +     -    6 0.50   5.7  638.4  1.1e-189 !   -
+
+contig--2255364      -         SSU_rRNA_bacteria    RF00177   hmm      325     1084      765        1      -     -    6 0.52   3.9  624.0  2.5e-185 !   -
+
+contig--2369838      -         SSU_rRNA_bacteria    RF00177   hmm       90      938       20      855      +     -    6 0.52   7.0  609.3  6.6e-181 !   -
+
+contig--2271497      -         SSU_rRNA_bacteria    RF00177   hmm        4      812      835        2      -     -    6 0.51   5.0  597.5  2.5e-177 !   -
+
+contig--2369838      -         SSU_rRNA_archaea     RF01959   hmm       70      894       17      855      +     -    6 0.52   5.4  346.7  9.9e-102 !   -
+
+**cmsearch.tblout.deoverlapped:**
+
+contig--2249925      -         SSU_rRNA_bacteria    RF00177   hmm       19      914        2      898      +     -    6 0.50   5.7  638.4  1.1e-189 !   -
+
+contig--2255364      -         SSU_rRNA_bacteria    RF00177   hmm      325     1084      765        1      -     -    6 0.52   3.9  624.0  2.5e-185 !   -
+
+contig--2271497      -         SSU_rRNA_bacteria    RF00177   hmm        4      812      835        2      -     -    6 0.51   5.0  597.5  2.5e-177 !   -
+
+contig--2369838      -         SSU_rRNA_bacteria    RF00177   hmm       90      938       20      855      +     -    6 0.52   7.0  609.3  6.6e-181 !   -
+
+
+    ]]></help>
+    <citations>
+        <citation type="doi">10.1093/nar/gkac1080</citation>
```

Comment:
Is the citation correct?

Reply:
No, this is the old wrong one. I just updated it.
Thank you for reminding me :)